UVicMartletplace/martletplace
# [Auth] Implementation for Python

# Description
Create the authentication middleware for Python

Closes #319 

## How to Test
Load the frontend and observe correct search/recommend functionality. Copy the request as curl and modify the JWT cookie. Observe the failing requests.

## Checklist
